elm-app/src/elm/Translate.elm

```diff
@@ -151,6 +160,23 @@ translate lang trans =
                     , hebrew = "חפשו אירועים לטעמכם"
                     }
 
+                HttpError stringId ->
```

Comment:
I know we are not using `HttpError` for now but I say let's just keep it in case we did some sort of lazy loading or something.